internal/js/modules/k6/browser/common/route.go

```diff
@@ -0,0 +1,73 @@
+package common
```

Comment:
I'm not sure about this refactoring. I'd keep all HTTP-related code in `http.go` as before, as I don't mind about the file line length. No strong opinion, though.

Reply:
I'll revert this change! Now that I think again about this, I think it makes sense to keep it in `http.go`. And if the file size becomes an issue, it would probably make more sense to split it into three to have `response.go`, `request.go` and `route.go`.

Reply:
❤️ In that case, I'd suggest `http_response.go`, `http_request.go`, etc.